### PR TITLE
feat(TBD-11035): Support MapR 6.1 with MEP 6.1.1 Static Lightweight Distro - DI / Spark Batch / Spark Streaming

### PR DESCRIPTION
**Pull request by mtuhai**
**Please check if the PR fulfills these requirements**

- [X] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [X] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [X] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [X] Docs have been added / updated (for bug fixes / features) ?
- [X] The new code does not introduce new technical issues

MapR 6.1.0 with MEP 6.1.1 lightweight distribution support

DO NOT MERGE

### CODE REVIEW

**Review by Redwene — APPROVED** [review]
lgtm

File: main/plugins/org.talend.hadoop.distribution.mapr611/.project [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		<buildCommand>
+			<name>org.eclipse.pde.ManifestBuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+		<buildCommand>
+			<name>org.eclipse.pde.SchemaBuilder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
```
Comment (ikulish) [inline main/plugins/org.talend.hadoop.distribution.mapr611/.project:22]: In org.talend.hadoop.distribution.mapr610/.project there is also 
```
		<buildCommand>
			<name>org.eclipse.m2e.core.maven2Builder</name>
			<arguments>
			</arguments>
		</buildCommand>
```
Do we need it here?

Comment (Redwene) [reply main/plugins/org.talend.hadoop.distribution.mapr611/.project:22]: this is a file for eclipse.

Comment (mtuhai) [reply main/plugins/org.talend.hadoop.distribution.mapr611/.project:22]: Yes, this is eclipse project. Does not impact maven build

**Review by CarlosTalend — APPROVED** [review]
LZ4 OK
Kite OK
Spark to 2.3.3.100-mapr-611 OK
Hive to mapr-spark-MEP-6.0.0-1904 OK
No misnamed findings for previous versions
LGTM

**Review by ikulish — APPROVED** [review]

**Pull request merged**